GETTING_STARTED.md

```diff
@@ -222,9 +222,9 @@ Howitzer provides 3 different validation types:
   </tr>
   <tr>
     <td>:element_presence</td>
-    <td>locator</td>
+    <td>name</td>
```

Comment:
How about rename this parameter 'name' to 'element' or 'element_name'?
Currently it looks like we checking presence of element by html attribute 'name'.